products/compute/inspec.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO(alexstephen): Match all objects w/ Puppet.
```

Comment:
I'm curious - does inspec want access to `puppet.yaml`? The Terraform OiCS/Example provider wants the same thing but for `terraform.yaml`, so we could consider a solution that encompasses both.

Reply:
Fixed! This was an old comment from when I copied this file from chef. I don't think there is a reason to hook this up with puppet. It will have all of its own resources by the end.